### DISCUSSION TITLE
How to get the default format?

### DISCUSSION THREAD

**Discussion by wenijinew** [starship/starship#6432] [General]
Hi,

I just set the starship and get the below prompt which looks not fully correct on virtualenv part.

`[USER NAME HERE] in 🌐 [HOST NAME HERE] in mllp on  main via 🐍 v3.12.0 (venv_v3.12.0'') `

As you see, there are two single quiotes after virtual env name, which should not be like that.

Do you know how to get the format string in default configuration. Withthat, I can tune it to fix.

Thanks!

**Comment by wenijinew** [comment]
I got it:
https://starship.rs/config/#default-prompt-format

**Reply by davidkna** [reply]
There is also `starship print-config --default format`.
